app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def index
+    @comments = Comment.all
+    render json: @comments
+  end
+
+  def create
+    @comment = Comment.new(
+      text: comment_params[:text],
+      user_id: comment_params[:users_id],
+      post_id: comment_params[:users_id]
+    )
+    if @comment.save
+      @comment.update_post_comments_counter
+      render json: @comment, status: :created
+    else
+      render json: { errors: @comment.errors.full_messages }, status: :unprocessable_entity
+    end
+  end
```

Comment:
![image](https://github.com/Megagig/Blog-App/assets/93039359/f62b60b0-98a7-4bf8-92c5-7ceea4136e76)

- [ ] Great work so far but I am still getting an error regarding the authenticity token :fearful: Kindly fix this issue and test your app well before submitting it for another review :+1: .